ibis/backends/bigquery/__init__.py

```diff
@@ -670,16 +749,14 @@ def _make_session(self) -> tuple[str, str]:
         return None
 
     def _get_schema_using_query(self, query: str) -> sch.Schema:
-        job = self.client.query(
+        job = self._client_query(
```

Comment:
The issue is here. `_get_schema_using_query` needs access to a `bq.QueryJob`, but `_client_query` always returns a `RowIterator`.

I'll fix this up!